css-brochure-form/styles.css

```diff
@@ -0,0 +1,75 @@
+body {
+  text-align: center;
+}
+
+h3 {
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  color: rgb(236 93 87);
+  width: 175px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+form {
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 0.8rem;
+  width: 500px;
+  margin: auto;
+}
+
+div.fieldtext {
+  margin-bottom: 3px;
+  margin-left: 37px;
+  text-align: left;
+}
+
+input::placeholder {
```

Comment:
Move padding to `.userinput`. That way when the user starts to type in the input field it starts at the same place as the placeholder text.